test/functional/tests/security/fuzzy/kernel/fuzzy_add_core/test_fuzzy_script_add_core_device_try_add.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+import random
+
+from api.cas.cli import script_try_add_cmd, remove_detached_cmd
+from core.test_run import TestRun
+from test_tools.peach_fuzzer.peach_fuzzer import PeachFuzzer
+from tests.security.fuzzy.common.common import global_fuzz_iterations_count
+from tests.security.fuzzy.kernel.common.common import get_fuzz_config, run_cmd_and_validate
+
+iterations_count = global_fuzz_iterations_count if global_fuzz_iterations_count else 1000
+
+
+def test_fuzzy_add_core_device_try_add():
```

Comment:
Should this test be parametrized (eg. start and stop cache to reload kernel modules)?

Reply:
Could you clarify what parameters you would like to add to the test?

Reply:
unaligned_io and use_io_scheduler in similar fashion as in test_fuzzy_start_cache_device line 41 (I guess other parameters don't matter in this case)